pkg/object/risingwave_manager.go

```diff
@@ -154,6 +154,16 @@ func (mgr *RisingWaveManager) IsOpenKruiseEnabled() bool {
 	return mgr.IsOpenKruiseAvailable() && risingwave.Spec.EnableOpenKruise != nil && *risingwave.Spec.EnableOpenKruise
 }
 
+func (mgr *RisingWaveManager) ReleaseLock(idx []int) {
+	mgr.mu.Lock()
+	defer mgr.mu.Unlock()
+
+	scaleViews := mgr.mutableRisingWave.Status.ScaleViews
+	for _, i := range idx {
+		mgr.mutableRisingWave.Status.ScaleViews = append(scaleViews[:i], scaleViews[i+1:]...)
+	}
+}
```

Comment:
I think we'll need to add a comment on this to let the developers know that
- This method could only be invoked once because the indexes are changed after a successful invocation.

Or shall we change the method to `KeepLocks` and set a slice of the alive lock records, which makes it reentrant?